test/commands-tests/multipleContractsFolder/ExampleContract8.aes

```diff
@@ -0,0 +1,20 @@
+contract Test =
```

Comment:
and for the tests you can use more descriptive contracts, like Math and sub-math contracts or Person, Student, Teacher to be more readable, because with example6,7,8 it is hard to navigate where you are